### ISSUE TITLE
Support 32-bit Windows

### ISSUE THREAD

**Issue by osworx** [brave/browser-laptop#397]
Latest Brave release, Win7 Ultimate 32bit
Installer starts, splash screen is show - that's it.
Final message: ```Installation has failed```

Log shows many entries, the most important seems to be that 32bit is not supported because several entries like ```System.ComponentModel.Win32Exception (0x80004005): Die angegebene ausführbare Datei ist keine gültige Anwendung für diese Betriebssystemplattform.``` are existing.

**Comment by bbondy** [comment]
Yep we only offer winx64 builds right now.

**Comment by kdpuvvadi** [comment]
What about Win32 users?

**Comment by osworx** [comment]
@bbondy thx, means brave is dead for me - and for about 60% of all users worldwide ..

**Comment by bbondy** [comment]
The stats are very different for our supported platforms (win7 and up).
@osworx note that I did not close this issue, I just said we don't have that yet.
We've only released a dev build on some platforms so far.

**Comment by PFCKrutonium** [comment]
@osworx 
![stats](https://cloud.githubusercontent.com/assets/3945538/12677501/78701a7e-c667-11e5-8b34-da9293d5b26a.PNG)

**Comment by bbondy** [comment]
Thanks @PFCKrutonium

**Comment by Mparaiso** [comment]
@PFCKrutonium It's still 10% using a 32 bit version of Windows ( Windows 7 for instance ) ,  1 out of 10 , so what does your message mean ? "the hell with these people" ? All browser vendors on Windows provide a 32 bit version of their product , even IE 11.

**Comment by bbondy** [comment]
We care about 32bit as well but we aren't offering builds for it yet.

There are some additional complications relating to packaging which we don't support yet to be able to do them.  There is also some ongoing incremental extra work to be able to offer those right now which we don't have the QA team size to support yet.

Note that x64 is more secure and more performant.

**Comment by SidepocketNeo** [comment]
Well this solves the issue on my end, I know that x64 is more secure and has better performance BUT as most will agree with me, I'm also not made out of money to get a new machine. More like two or three years from now I can afford one. Are the Linux versions locked at x64 as well because then I can dual boot.

**Comment by PFCKrutonium** [comment]
How old is your computer? Anything that is less than a decade old can run x64 Windows.

**Comment by osworx** [comment]
@PFCKrutonium Funny talking here ..
Brave wants to earn money through the replacements of adverts - their business.
If they do not support 32bit installations, they will loose money - their business.
When I (and some others) are using 32bit machines / installations - my business, and I will have reasons to do so.
Has nothing to do how old are our computers.

What you are running, and thinking - again your business.
But keep such for yourself.

**Comment by bbondy** [comment]
tldr there are some good reasons for people to use 32bit. We don't support it yet though.
I think all points are clearly communicated now.  But will lock from comments if needed to avoid excess spam. Thanks.

**Comment by bridiver** [comment]
I think you might be misunderstanding our position @osworx. We are not currently supporting 32bit, but that does not mean we will never support 32bit. These are still dev releases and 32bit support is not a priority at the moment. In the meantime we would be more than happy to look at any pull requests for 32bit build/packaging support :)

**Comment by Ken-Saunders** [comment]
I hope that y'all do provide 32-bit versions eventually and not alienate anyone. 
It does go without saying but I'm saying anyway that Brave of course can't afford to with the field that its in.
It's probably easier to enter into the cola market and be successful then the browser one. 
I think that it's brave that, oh, wait, Brave, ah ok. Just kidding and trying to lighten it up.

I'm using a PC less than a decade old running 32-bit Windows 7 with the latest 32-bit versions of Firefox, the " cutting edge" Nightly, Thunderbird, Daily, and so on and I hardly feel as if I'm behind the times using old tech. 

While I'm not a metrics expert, in my opinion and experience, you have to be very careful here and not always go by raw stats. I see Mozilla making that mistake with some to a lot aspects with Firefox and they're pissing off and losing the early adopters types and power users. The ones not afraid to try anything, dig in, contribute, stay loyal, and help build. Many are looking for an out but there isn't a suitable one. 
Brendan and you bbondy are aware of that.
For the record, I'm not one of those.

If I didn't have such respect and admiration for Brendan and wanting to see him succeed, and if I weren't so anxious to see more competition against Google, I would have moved on the moment after having my first issues with Brave as I did.
Not many others would/will stick with it ya' know.

Hopefully issue <a href="">#80</a> will avoid all of this frustration and confusion in the future.

**Comment by Ken-Saunders** [comment]
Wrong link
https://github.com/brave/brave-website/issues/80

**Comment by hoangtronghuy20** [comment]
![c](https://cloud.githubusercontent.com/assets/13413419/13373595/6c323d16-dd9e-11e5-9ba7-ecd05daccc89.JPG)

**Comment by PFCKrutonium** [comment]
@hoangtronghuy20 Did you bother to read this issue? It will be fixed eventually, but not yet.

**Comment by bbondy** [comment]
Good news, 0.9.2 is released and it has 32-bit Windows installers too.
https://github.com/brave/browser-laptop/releases/tag/v0.9.2dev

**Comment by tukoz** [comment]
About #397 does it build on != Windows platforms?

**Comment by SidepocketNeo** [comment]
So happy this is now on 32 platform. Running it, will be looking for bugs. Thanks!

**Comment by tukoz** [comment]
Howdy! and I can't wait to read @osworx feedback. To my understanding team deserves it since they *did* as they said they would 👍
